### PR TITLE
JAMES-1676 Fix Eclipse compilation & remove warnings

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/UpdateMessagePatchValidatorTest.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 package org.apache.james.jmap.methods;
 
-import static org.assertj.core.api.Assertions.assertThat;;
```
Comment (aduprat) [inline server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/UpdateMessagePatchValidatorTest.java]: This was the compilation error.

**Comment by rouazana** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**